win/CS/HandBrakeWPF/Helpers/AutoNameHelper.cs

```diff
@@ -86,6 +87,35 @@ public static string AutoName(EncodeTask task, string sourceOrLabelName, Preset
                 if (chapterFinish != chapterStart && chapterFinish != string.Empty)
                     combinedChapterTag = chapterStart + "-" + chapterFinish;
 
+                // Local method to check if we have a creation time in the meta data. If not, fall back to source file creation time.
+                DateTime obtainCreateDateObject()
+                {
+                    var rd = task.MetaData.ReleaseDate;
+                    if (DateTime.TryParse(rd, out var d))
+                    {
+                        return d;
+                    }
+                    try
+                    {
+                        return File.GetCreationTime(task.Source);

```

Comment:
Have you tested this with a DVD source where you only have a path not a filename?  I'm worried it might trigger an exception to get thrown below.

Reply:
ah I actually completely forgot to try a dvd input!
- [ ] Test this this out with dvd input; fix any issues that may arise

Reply:
I tested with a DVD source and had no issues and looks like it picked up the date / time of the disk (in my case putting a breakpoint on it I did verify the source was just "G:\")